priv/repo/migrations/20221129193311_alter_mentors.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Bokken.Repo.Migrations.AlterMentors do
+  use Ecto.Migration
+
+  def change do
+    alter table(:mentors, primary_key: false) do
+      add :t_shirt, :string
```

Comment:
```suggestion
      add :t_shirt, :citext
```
This way the value is case insensitive. It shouldn't really matter, but I think it makes more sense.
(Citext is a Postgresql extension)